x-pack/plugins/integrations_manager/public/screens/home.tsx

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import React, { useState, useEffect } from 'react';
+import { EuiFlexGrid, EuiFlexItem, EuiPanel, EuiSpacer, EuiText, EuiTitle } from '@elastic/eui';
+import { getIntegrationsList } from '../data';
+import { IntegrationCard } from '../components/integration_card';
+import { IntegrationList } from '../../common/types';
+
+export function Home() {
+  const [list, setList] = useState([] as IntegrationList);
```

Comment:
To avoid the type assertion we should do:

```suggestion
  const [list, setList] = useState<IntegrationList>([]);
```